Cargo.toml

```diff
@@ -27,12 +27,13 @@ imgui-winit-support = { version = "0.2.0", default-features = false, features =
 log = { version = "0.4.8" }
 nalgebra = { version = "0.19.0", features = ["serde-serialize"] }
 png = "0.15.0"
-serde = "1.0.102"
+serde = { version = "1.0.102", features = ["derive", "rc"] }
```

Comment:
Meh. The `rc` features may come to bite us later. I suggest the following:
- Remove the `rc` feature
- Change `LitExpr::String` to contain just `String`, not `Arc<String>`
- Fix the resulting compile errors by:
  - When evaluating a `LitExpr` to a `Value`, just clone the string in `interpreter/mod.rs`.
  - Remove the completely needless 3 `Arc::new` calls in `ui.rs`

Btw even if the docs say otherwise, the `derive` features seems to be a enabled by default. We already have some `derive(serde::Serialize)` in the codebase and they work just fine.